api/python/cellxgene_census/tests/experimental/ml/test_pytorch.py

```diff
@@ -419,6 +421,7 @@ def test_distributed__returns_data_partition_for_rank(
             measurement_name="RNA",
             X_name="raw",
             obs_column_names=["label"],
+            encoders=[DefaultEncoder("soma_joinid"), DefaultEncoder("label")],
```

Comment:
These test use the `soma_joinid` to assert positional conditions, so this is how you force the `soma_joinid` to be part of the encoder.

Reply:
Ah, then this point really should be explained in the docstring for `encoders` param.

Reply:
Also would `obs_column_namess=["soma_joinid", "label"]` be equivalent?

Reply:
Not the same - `soma_joinid` is ignored because by default only string columns are encoded when using the default behavior. Using custom encoders override this.